amoro-web/src/language/en.ts

```diff
@@ -127,8 +127,8 @@ export default {
   resourceGroup: 'Resource Group',
   releaseOptModalTitle: 'Release this optimizer？',
   cancelOptimizingProcessOptModalTitle: 'Cancel the optimizing process of this table？',
```

Comment:
There are four places where ? is a full-width character.

Reply:
Yes, we should replace the question mark in Chinese(？) with the English one(?).